crates/tabby-common/src/path.rs

```diff
@@ -53,4 +53,8 @@ pub fn events_dir() -> PathBuf {
     tabby_root().join("events")
 }
 
+pub fn background_jobs_dir() -> PathBuf {
+    tabby_root().join("background_jobs")
+}
+
```

Comment:
should use https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/main/ee/tabby-webserver/src/path.rs?plain=1#L5

tabby_ee_background_jobs_dir()